### DISCUSSION TITLE
也谈苏东坡 - Qinng

### DISCUSSION THREAD

**Discussion by giscus** [qingwave/qingwave.github.io#71] [Announcements]
# 也谈苏东坡 - Qinng

但少闲人如吾两人尔李一冰的《苏东坡新传》在微信读书里躺了一年有余，偶尔想起翻开几页，不过大多时候是想不起的。年纪越大越发的浮躁，少了上学时的心平气和，工作之后阅读的功利心重了许多，总想通过几段文字掌握

https://qingwave.github.io/sudongpo/

**Comment by Wangmin362** [comment]
也无风雨也无晴
